### DISCUSSION TITLE
Different Support x/y distance for interface layer(s)

### DISCUSSION THREAD

**Discussion by ADKMechETECH** [OrcaSlicer/OrcaSlicer#5568] [Ideas]
I find myself printing flat bottom surfaces (with normal supports) and very small or zero top z distance.  This is working very well and the bottom surfaces of the parts look fantastic.  I am currently printing PET-CF with what I believe is probably actually HIPS support material.

The issue I have is that when I have sharp inside corners on supported overhangs, the model material has to bridge this small x/y gap and extrudes down past the support interface layer a bit.  This leaves a fuzzy non-sharp inside corner.  With the PET-CF I need to have at least 0.2mm x/y clearance, but I would prefer to have 0.3-0.4mm.  

Is there a possibility to have a separate x/y setting for only the interface layers?  I have found that 3 interface layers works well and a setting for % overlap per layer would be helpful to prevent printing unsupported.  For instance, If you want to expand the very top interface layer to 0 x/y and have a 0.3mm x/y for support base, you could specify 0.1mm step over per interface layer so you have 75% overlap using a 0.4mm nozzle and 0.4mm extrusion width. 

Is this crazy, does anyone else think this would be useful?  Or, am I missing a setting that would already do this?

**Comment by brysonhyte** [comment]
This is a slightly different use case, but the UI options remain the same. Using mixed materials between the supports and the main body (e.g., PLA/PETG) impacts tool changes and increases print time. Provide the ability to specify a different material for the Support Raft/Interfaces—both top and bottom—as well as the support body. This would help minimize tool changes by allowing the same material to be used for both the support and main body, ultimately reducing print time.

Another important consideration is that when printing supports with an alternate material, the Support Object X/Y distance may need to be adjusted based on the material. For example, when using PLA/PETG for supports, the support settings typically have a "zero" distance since they don’t bond. However, if you choose to use the same material for both the support body and the main body to reduce tool changes and print time, the offset distance may be too close, causing the materials to bond unintentionally.
